### DISCUSSION TITLE
Incorporating Huggingface BART model into Spark NLP

### DISCUSSION THREAD

**Discussion by ericbugin** [JohnSnowLabs/spark-nlp#7167] [Q&A]
Hello, is there a way to get this model into a spark pipeline using Spark NLP? I know we can use Spark NLP to download particular models from HF, but was wondering if using BART models like this was doable. Thanks
```
from transformers import pipeline
classifier = pipeline("zero-shot-classification",
                      model="facebook/bart-large-mnli")
```

**Comment by maziyarpanahi** [comment]
Hi,
You can only import models from TF Hub and HuggingFace for the equivalent annotators we have. The current compatible list and instruction is here https://github.com/JohnSnowLabs/spark-nlp/discussions/5669

BART can be a feature request, I think I saw it somewhere this week.
